### DISCUSSION TITLE
service/screen share not auto starting after reboot on Android

### DISCUSSION THREAD

**Discussion by plia7** [rustdesk/rustdesk#4691] [Q&A]
I got the following phone:

Moto G Pure | 2021 | 2-Day battery | Unlocked | Made for US by Motorola | 3/32GB | 13MP Camera | Deep Indigo

[https://www.amazon.com/gp/product/B09BTB5MKT/](https://www.amazon.com/gp/product/B09BTB5MKT/)

It has an Android 11 installed.

After installing RuskDesk Version: 1.1.10-1
and granting it the following 4 permissions:

Screen Capture, Input Control, Transfer File and Audio Capture

I was able to connect using the generated id and password and everything was working fine.

After rebooting my phone, I was not able to reconnect anymore. When I opened the app and went to the screen share tab, this what I saw there:

“A Service is not running Tap [Start Service] or OPEN [Screen Capture] permission to start the screen sharing service.”

All the other 3 permissions were already OPEN, it’s just the screen capture that wasn’t.

Is there a setting, adb permissions command or another way to make it automatically start the service/screen capture just like how TeamViewer Host does it? 

I found this discussion on github page:

https://github.com/rustdesk/rustdesk/discussions/1516

I found in the end of that post they implemented this as a feature here:

https://github.com/rustdesk/rustdesk/pull/3425

But I don’t see an option anywhere in the RustDesk app where to enable start on boot setting assuming this feature already exists in the current version that I’m using? Do you have this option on your app?

I also try to give it the following adb permission:

```
adb shell cmd appops set com.carriez.flutter_hbb PROJECT_MEDIA allow
```
But I don’t see evidence it’s being granted when running:
```
adb shell dumpsys package com.carriez.flutter_hbb
```

Thank you.

**Comment by ares1977** [comment]
You have to install the nightly version

**Reply by plia7** [reply]
> You have to install the nightly version

Thanks, is there an apk link for the nightly version?

**Comment by deep-soft** [comment]
1.2.0 [Latest](https://github.com/rustdesk/rustdesk/releases/latest)
@[rustdesk](https://github.com/rustdesk) rustdesk released this yesterday

https://github.com/rustdesk/rustdesk/releases/tag/1.2.0
https://github.com/rustdesk/rustdesk/releases/download/1.2.0/rustdesk-1.2.0-aarch64-signed.apk

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#android-start-on-boot-and-limitations

any further discussion please go to https://github.com/rustdesk/rustdesk/discussions/1330
